src/server/scripts/EasternKingdoms/ZulGurub/boss_jeklik.cpp

```diff
@@ -97,57 +122,86 @@ Position const SpawnBat[6] =
     { -12293.6220f, -1380.2640f, 144.8304f, 5.483f }
 };
 
-enum Misc
+Position const SpawnBatRider = { -12301.689, -1371.2921, 145.09244 };
+Position const JeklikCaveHomePosition = { -12291.9f, -1380.08f, 144.902f, 2.28638f };
+
+enum PathID
 {
-    PATH_JEKLIK_INTRO = 145170
+    PATH_JEKLIK_INTRO = 145170,
+    PATH_BAT_RIDER_LOOP = 147500
```

Comment:
```suggestion
    PATH_JEKLIK_INTRO   = 145170,
    PATH_BAT_RIDER_LOOP = 147500
```
Enums out of line with each other.